pkg/certificate/options.go

```diff
@@ -20,19 +18,6 @@ func (o *issueOptions) formatCN(prefix, trustDomain string) CommonName {
 	return CommonName(fmt.Sprintf("%s.%s", prefix, trustDomain))
 }
 
-func defaultOptions(validityPeriod time.Duration) *issueOptions {
-	return &issueOptions{
-		validityPeriod: validityPeriod,
-	}
-}
-
-// WithValidityPeriod modifies the expiration on a newly issued certificate.
-func WithValidityPeriod(validityPeriod time.Duration) IssueOption {
```

Comment:
@steeling I removed this because I didn't want to make the cert type/ validity duration optional and assume a default. With my current design for cert rotation and issuing it is required that I have a cert type or something similar to determine whether to call getServiceCertValidityDuration, getIngressGatewayCertValidityDuration, or use the OSM internal cert duration. Do you have any suggestions for how to incorporate the validity period options with my changes or take a different approach? Thanks!!